### DISCUSSION TITLE
Get Traffic Spike Alerts through a webhook

### DISCUSSION THREAD

**Discussion by NathanC** [plausible/analytics#2378] [Anything Else]
Hello! I'm trying to build a comprehensive budget alert system, and having traffic spike alerts from plausible would be a very useful early warning sign.

Right now, it looks like they can be sent through email or slack. What about adding a generic webhook? It seems like it would be very easy to add—just have a registered webhook, and invoke it at the same time as the current traffic spike alerts are sent. The relevant info could be included in a JSON body.

I could then have my webhook send me a text, or hook into pager duty, or degrade the site if traffic is way too high, etc etc.

Along with this, having some more options for traffic spike alerts (having 2 thresholds for example?) would be very useful.

Thoughts?

**Comment by susanthenerd** [comment]
I would love to see this implemented as well. It would be very practical for me to receive a notification over webhooks.

**Comment by lloydjatkinson** [comment]
I also would like this! Instead of needing to poll the stats API we could have the same stats pushed to us instead.
